src/System.Data.SqlClient/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,7 @@
+MembersMustExist : Member 'System.Data.SqlClient.SqlBulkCopy.WriteToServer(System.Data.Common.DbDataReader)' does not exist in the implementation but it does exist in the contract.
```

Comment:
@ericstj These contracts should move to 4.5.1. 
I confirmed that System.Data.Common minimum compatibility should be with 4.5.1